pkg/kubectl/cmd/patch.go

```diff
@@ -245,14 +245,8 @@ func (o *PatchOptions) RunPatch() error {
 			if err != nil {
 				return err
 			}
-			printer.PrintObj(info.Object, o.Out)
```

Comment:
Can you please add a tests verifying:
1. that exit code is 0, when the patch was empty
2. that exit code is non-zero, when the patch command actually failed

Reply:
@soltysh Added a test. Although I realize after adding it, that only semantic or type errors will result in the error being printed out and an exit code of 1. If someone were to, for example, update an immutable field on a pod, or set a value for a non-existent spec field on a pod (like replicas), the server would just return the old object with no error, and the command would exit with the message "not patched" (as before), but with exit code 0 (since the old and new object were the same)

Reply:
That's reasonable.